packages/chat/components/chartElement/__stories__/chart.mdx

```diff
@@ -1,4 +1,4 @@
-import { ArgTypes, Markdown, Meta } from '@storybook/blocks';
+import { ArgTypes, Markdown, Meta, Controls, ArgsTable } from '@storybook/blocks';
```

Comment:
Don't think we would need `Controls` or `ArgsTable` if we're using `ArgTypes`

Reply:
Alright!